build_package.py

```diff
@@ -1234,6 +1233,31 @@ def create_scriptlets():
     fp.write(
         r"""#!/bin/bash
 
+echo "Check for python."
+# look for 'python' by '/usr/bin/env', exit with error if it is not found.
+/usr/bin/env python --version >/dev/null 2>&1 || \
+{ echo -e >&2 "\e[31mThe suitable python interpreter is not found. Aborting.\e[0m"; \
+echo -e >&2 "The command - '/usr/bin/env python' has to point to the python interpreter with version \e[31m>=2.6 or >=3.5\e[0m."; exit 1; }
```

Comment:
Perhaps we could provide a bit more user-friendly error message here and instruct user to install ``python`` / ``python2`` package if that binary is not available.

We don't need to go overboard with the distro detection, just a simple detection if it's Debian or RHEL based system and display either apt-get or yum command show work.